composer.json

```
@@ -21,7 +21,7 @@
     }
   },
   "require-dev": {
-    "phpunit/phpunit": "^4.8.36 || ^5.7.27 || ^7.3",
+    "phpunit/phpunit": "^5.7.27 || ^7.3 || ^8 || ^9",
```

```suggestion
    "phpunit/phpunit": ">=5.7.27",
```
Should be equivalent

---

But would be a very bad idea ;)
It would silently allow next major 10
I would agree for `>=5.7.27 <11`

---

My bad :+1:  but maybe <10 than <11